### PR TITLE
[Doc] Add missing docs for the coming version 1.1.1

### PR DESCRIPTION
**Pull request by banmoy**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [X] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
- [ ] I have added documentation for my new feature or new function

### CODE REVIEW

File: docs/connector-write.md [hunk 1]
```diff
@@ -75,15 +82,18 @@ Directly download the corresponding version of the Spark connector JAR from the
...
 | starrocks.write.label.prefix                   | NO       | spark-        | The label prefix used by Stream Load.                        |
-| starrocks.write.enable.transaction-stream-load | NO       | TRUE          | Whether to use [the transactional interface of Stream Load](https://docs.starrocks.io/en-us/latest/loading/Stream_Load_transaction_interface) to load data. This feature is supported in StarRocks v2.4 and later. |
+| starrocks.write.enable.transaction-stream-load | NO       | TRUE          | Whether to use [the transactional interface of Stream Load](https://docs.starrocks.io/en-us/latest/loading/Stream_Load_transaction_interface) to load data. It requires StarRocks v2.5 or later. This feature could load more data in a transaction with less memory usage, and improve performance. <br/> **NOTICE:** Since 1.1.1, this option is only available if `starrocks.write.max.retries` is non-positive, that's, transaction stream load does not support retry.                                                                                                                                                                                                                                 |
```
Comment (hellolilyliuyi) [inline docs/connector-write.md:91]: 如果是streamload事务接口 一次性发送的数据
截屏2023-09-01 下午2 18 58
Stream Load 事务接口允许在一个导入作业中按需合并发送多次小批量的数据后“提交事务”--这个是flink connector后台操作的吗？不需要用户设置吗？这里面遵循的逻辑是啥？

File: docs/connector-read.md [hunk 2]
```diff
@@ -235,7 +240,13 @@ The processing logic of the underlying storage engine used by StarRocks cannot c
 
 ## Spark connector upgrades
 
-### Upgrade from version 1.0.0 to version 1.1.0
+### Upgrade from version 1.1.0 to 1.1.1
+
+* Since 1.1.1, the connector does not provide `mysql-connector-java` which is the official JDBC driver for MySQL. It uses GPL license which has some limitations.
```
Comment (hellolilyliuyi) [inline docs/connector-read.md:245]: MySQL JDBC driver 似乎一直用gpl协议为啥现在要用户自己手动下载，gpl限制是啥

File: docs/connector-write.md [hunk 3]
```diff
@@ -116,13 +127,21 @@ Directly download the corresponding version of the Spark connector JAR from the
...
 
+## Spark connector upgrades
 
-- You can also customize the data type mapping.
+### Upgrade from version 1.1.0 to 1.1.1
+
+* Since 1.1.1, the connector does not provide `mysql-connector-java` which is the official JDBC driver for MySQL. It uses GPL license which has some limitations.
+  The connector needs the JDBC driver to visit StarRocks for the metas of tables, so you need add the driver to the spark classpath manually. You can find the
+  driver on [MySQL site](https://dev.mysql.com/downloads/connector/j/) or [Maven Central](https://repo1.maven.org/maven2/mysql/mysql-connector-java/).
+* Since 1.1.1, the connector will use general stream load by default rather than transaction stream load in version 1.1.0. If you want to go back to transaction stream load, you
```
Comment (hellolilyliuyi) [inline docs/connector-write.md:141]: 为啥flink推荐用事务接口 但是spark推荐用非事务接口？

Comment (banmoy) [reply docs/connector-write.md:141]: from the feedback of the users, they think the retry is more important to make the spark stable, but transaction stream could not support retry, so disable it by default

File: docs/connector-read.md (hunk 2 above)
Comment (banmoy) [reply docs/connector-read.md:245]: GPL is an open source license, and has some limitations for the free use of the software. https://en.wikipedia.org/wiki/GNU_General_Public_License

File: docs/connector-write.md (hunk 1 above)
Comment (banmoy) [reply docs/connector-write.md:91]: yes, the connector will split the data in one load to multiple batches

File: docs/connector-read.md [hunk 4]
```diff
@@ -161,6 +165,7 @@ The following parameters apply to all three reading methods: Spark SQL, Spark Da
 | starrocks.deserialize.arrow.async    | false             | Specifies whether to support asynchronously converting the Arrow memory format to RowBatches required for the iteration of the Spark connector. |
 | starrocks.deserialize.queue.size     | 64                | The size of the internal queue that holds tasks for asynchronously converting the Arrow memory format to RowBatches. This parameter is valid when `starrocks.deserialize.arrow.async` is set to `true`. |
 | starrocks.filter.query               | None              | The condition based on which you want to filter data on StarRocks. You can specify multiple filter conditions, which must be joined by `and`. StarRocks filters the data from the StarRocks table based on the specified filter conditions before the data is read by Spark. |
+| starrocks.timezone                   | Default timezone of JVM | Supported since 1.1.1. The timezone used to convert StarRocks `DATETIME` to Spark `TimestampType`. The default is the timezone of JVM returned by `ZoneId#systemDefault()`. The format could be a timezone name such as `Asia/Shanghai`, or a zone offset such as `+08:00`. |
```
[the remark this answers is not in the document]
Comment (banmoy) [reply docs/connector-read.md:168]: emm, it's better to have this column, and we can optimize it in another pr

**Pull request merged**